redaxo/src/addons/structure/lib/navigation.php

```diff
@@ -48,12 +52,12 @@ class rex_navigation
 
     private function __construct()
     {
-        // nichts zu tun
+        $this->activeLiClass = $this->activeClass;
+        $this->activeAClass = $this->activeClass;
+        $this->currentLiClass = $this->activeClass;
+        $this->currentAClass = $this->activeClass;
```

Comment:
`$this->activeClass` wird aber doch nur im Konstruktor ausgelesen. Wenn ich dann anschließen `$nav->setActiveClass()` aufrufe, ändere ich zwar theoretisch den Default-Wert im aktuellen Nav-Objekt, aber der Default-Wert wird gar nicht mehr ausgelesen werden.